src/components/EventForm/index.js

```diff
@@ -130,34 +135,67 @@ export default function EventForm({
     </div>
   );
 
+  //Function to save the input Title value when it changes
+  const onInput = (e) => {
+    setValueTitle(e.target.value)};
+
+  //Function to save the input Date-value when it changes
+  const onChange = (date, dateString) => {
+      if(dateString !== ""){
+        setValueDate(dateString)
+      }
+  };
+
+
   return (
     <Modal
       open={isModalOpen}
-      onOk={() => {
-        onFinish(form.getFieldsValue());
-      }}
       onCancel={() => {
         closeModal();
       }}
-      okText={"Save"}
       confirmLoading={isLoading}
       centered
+      footer={[
+        <Button key="cancelButton" onClick={()=>{closeModal()}}>Cancel</Button>,
+        <Tooltip key="saveTooltip" placement="top" title="Save available when title and date are filled in" >
+          <Button key="saveButton"type="primary" onClick={() =>{onFinish(form.getFieldsValue());}}
+                disabled={valueDate !== "" && valueTitle !== "" ? false : true}  
```

Comment:
Instead of maintaining two states, we can get the values by using form.getFieldValue() provided by antd.

Can you please check @Blueclaus13